### DISCUSSION TITLE
Sentry error

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [DuCanhGH/next-pwa#154] [Q&A]
Hi! I keep getting this error `wrsParams.serviceWorkers.navigator.serviceWorker.register` from google bot mostly; is there a way to add some kind of try catch or to handle this kind of errors during sw registration? Otherwise I have to filter this events, but I was wondering if it was possible to handle this cases, just to be sure that other users don't have this kind of problems.

Thanks!

**Comment by DuCanhGH** [comment]
@Emiliano-Bucci sorry for having left you waiting for two weeks :( It's pretty hard to keep track of things when GitHub doesn't notify me of new Discussions.

Regarding the issue, I believe that this is not possible with the current implementation of `@ducanh2912/next-pwa`. This is because our `sw-entry.ts` calls `Workbox.register()` without handling its errors.

What causes the function to throw though? This should not happen, unless there's an issue with the generated service worker.
